tools/gulp/tasks/payload.ts

```diff
@@ -25,10 +26,10 @@ task('payload', ['material:clean-build'], async () => {
     material_fesm_2015: getBundleSize('material.js'),
     material_fesm_2014: getBundleSize('material.es5.js'),
     // CDK bundles
-    cdk_umd: getBundleSize('cdk.umd.js'),
-    cdk_umd_minified_uglify: getBundleSize('cdk.umd.min.js'),
-    cdk_fesm_2015: getBundleSize('cdk.js'),
-    cdk_fesm_2014: getBundleSize('cdk.es5.js'),
+    cdk_umd: getBundleSize('cdk*.umd.js'),
+    cdk_umd_minified_uglify: getBundleSize('cdk*.umd.min.js'),
+    cdk_fesm_2015: getBundleSize('cdk.js') + getBundleSize('cdk/!(*.es5).js'),
+    cdk_fesm_2014: getBundleSize('cdk.es5.js') + getBundleSize('cdk/**.es5.js'),
```

Comment:
Is the pattern here missing a slash (e.g. `cdk/**/*.es5.js`)? Otherwise the second * seems redundant.